packages/core/src/index.js

```diff
@@ -8,7 +8,7 @@ export { Box } from './Box'
 export { Breadcrumbs } from './Breadcrumbs'
 export { Button } from './Button'
 export { Card } from './Card'
-export { Chip } from './Chip'
```

Comment:
If we remove Chip, this will need to be released as DS v5. It might be best to also export `FilterChip` as `Chip` to avoid breaking changes

Reply:
@ryan-hunt-priceline is the new `FilterChip` identical in functionality to the old `Chip`? It's probably worth a discussion, but we might want to keep the old `Chip` around, make this change purely additive, and mark the old `Chip` for deprecation in v5. Otherwise, we end up with inconsistent naming between the Chip variants.

Reply:
Yeah, for now I can export the FilterChip as Chip so it doesn't break current functionality

Reply:
Updated to export `FilterChip` as default of `Chip`